st2common/st2common/runners/base.py

```diff
@@ -227,12 +188,6 @@ def cancel(self):
             self.liveaction.context
         )
 
-    def post_run(self, status, result):
```

Comment:
Need to keep this function:
https://travis-ci.org/github/StackStorm/st2/jobs/755265427#L1555
https://app.circleci.com/pipelines/github/StackStorm/st2/872/workflows/c7ffafaa-d8b7-40b0-825e-61c36b404168/jobs/12202/parallel-runs/0/steps/0-110

Reply:
Thanks, will do. Will proceed on this PR on ~40 mins.

Reply:
Done.